compose/release-notes.md

```diff
@@ -10,6 +10,23 @@ redirect_from:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.20.2
+{% include release-date.html date="2023-07-19" %}
+
+### Bug fixes and enhancements
+- Add support of `depends_on.required` attribute
+- Fixed an issue about build trying to push unnamed service images
+- Fixed a bug with target secret path on Windows
```

Comment:
```suggestion
- Fixed a bug with target secret path on Windows
- Fixed a bug resolving build context path for services using `extends.file`
```

That fix comes from `compose-go` but was reported as docker/for-mac#6912, so I think might be worth an explicit callout.